cdap-data-fabric/src/main/java/co/cask/cdap/data2/nosql/NoSqlStructuredTable.java

```diff
@@ -178,16 +190,10 @@ public void close() throws IOException {
    * Convert the columns to corresponding byte array, each column has to be part of the schema.
    *
    * @param columns columns to convert
-   * @return the converted byte array, null if read for all columns
+   * @return the converted byte array
    * @throws InvalidFieldException some column is not part of the schema
    */
-  @Nullable
-  private byte[][] convertColumnsToBytes(@Nullable Collection<String> columns) throws InvalidFieldException {
-    // Empty columns means to read all columns. The corresponding parameter for Table is null
```

Comment:
is this an unrelated fix? 
I am not sure I understand why it is not nullable?

Reply:
In a previous PR, I had refactored the read method for all columns to not use this method. So null would never get passed to this method anymore.

Reply:
ok got it